integration-tests/jest/jest-utils.js

```diff
@@ -0,0 +1,21 @@
+const { screenshotDir } = require('../utils/integration-config')
+
+
+const defaultTimeout = 5 * 60 * 1000
+
+const withGlobalJestPuppeteerContext = test => async () => await test({ context, page: await context.newPage() })
```

Comment:
I think `jest-puppeteer` creates a default page and exposes a `page` global. Can we pass that along instead of creating another one?

Reply:
Yes, good catch. This was left over from putting all of the tests in a single file and running them concurrently. Can also switch back from `test.concurrent` to just `test` below.